.github/workflows/python_tests.yml

```diff
@@ -0,0 +1,33 @@
+name: python_tests
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Also add these to the javascript one; we need those before we remove the travis file, otherwise PRs won't get tests for JS

Reply:
Done in #4248